src/prommis/uky/tests/test_uky_flowsheet.py

```diff
@@ -211,133 +211,136 @@ def test_solution(system_frame):
         0, "Al2O3"
     ].value == pytest.approx(0.233476, 1e-4)
     assert model.fs.leach.solid_outlet.mass_frac_comp[0, "CaO"].value == pytest.approx(
-        0.0017975, 1e-4
+        0.0017995, 1e-4
```

Comment:
All these test changes are a result of the change made in L1129 of the UKy flowsheet: `m.fs.sl_sep2.liquid_recovery.fix(0.95)` instead of `m.fs.sl_sep2.liquid_recovery.fix(0.88)`

Reply:
Was there a specific reason this change was made?

Reply:
The recovery should've been 95% the entire time, but when I originally made the flowsheet, I was only able to get a recovery of 88%. This was marked as a TODO item in the flowsheet, but I had just forgotten about it until now.